src/overlays/actors/ovl_Koinobori/ac_koinobori.c

```diff
@@ -145,7 +145,7 @@ void aKOI_actor_draw(Actor* thisx, Game_Play* game_play) {
         gSPSegment(__polyOpa++, 0x0A, Lib_SegmentedToVirtual(&aKOI_obj_e_koinobori_a_pal));
         gSegments[6] = OS_K0_TO_PHYSICAL(object);
         gSPSegment(__polyOpa++, 0x06, object);
-        CLOSE_POLY_OPA_DISPS();
+        CLOSE_POLY_OPA_DISPS()
```

Comment:
I think the `;` styling is preferred

Reply:
This is actually an error I have no idea why this happened im trying to fix the PR right now

Reply:
It seems like format.py is automatically removing these semicolons